docs/userguide/storagedriver/aufs-driver.md

```diff
@@ -10,184 +10,203 @@ parent = "engine_driver"
+layer. So, the CoW must *copy up* the file from the underlying image. The AUFS
+storage driver searches each image layer for the file. The search order is from
+ top to bottom. When it is found, the entire file is *copied up* to the
+container's top writable layer. From there, it can be opened and modified.
+
+Larger files obviously take longer to *copy up* than smaller files, and files
+that exist in lower image layers take longer to locate than those in higher
+layers. However, a *copy up* operation only occurs once per file on any given
+container. Subsequent reads and writes happen against the file's copy already
+*copied-up* to the container's top layer.
 
 ## Deleting files with the AUFS storage driver
 
 The AUFS storage driver deletes a file from a container by placing a *whiteout
 file* in the container's top layer. The whiteout file effectively obscures the
-existence of the file in image's lower, read-only layers.  The simplified
+existence of the file in the read-only image layers below. The simplified
 diagram below shows a container based on an image with three image layers.
 
 ![](images/aufs_delete.jpg)
 
 The `file3` was deleted from the container. So, the AUFS storage driver  placed
 a whiteout file in the container's top layer. This whiteout file effectively
-"deletes"  `file3` from the container by obscuring any of the original file's
-existence in the image's read-only base layer. Of course, the image could have
-been in any of the other layers instead or in addition depending on how the
-layers are built.
+"deletes" `file3` from the container by obscuring any of the original file's
+existence in the image's read-only layers. This works the same no matter which
+of the image's read-only layers the file exists in.
 
 ## Configure Docker with AUFS
 
-You can only use the AUFS storage driver on Linux systems with AUFS installed. Use the following command to determine if your system supports AUFS.
+You can only use the AUFS storage driver on Linux systems with AUFS installed.
+Use the following command to determine if your system supports AUFS.
 
-```bash
-$ grep aufs /proc/filesystems
-nodev   aufs
-```
+    $ grep aufs /proc/filesystems
+    nodev   aufs
 
-This output indicates the system supports AUFS.  Once you've verified your
+This output indicates the system supports AUFS. Once you've verified your
 system supports AUFS, you can must instruct the Docker daemon to use it. You do
 this from the command line with the `docker daemon` command:
 
-```bash
-$ sudo docker daemon --storage-driver=aufs &
-```
+    $ sudo docker daemon --storage-driver=aufs &
+
 
 Alternatively, you can edit the Docker config file and add the
 `--storage-driver=aufs` option to the `DOCKER_OPTS` line.
 
-```bash
-# Use DOCKER_OPTS to modify the daemon startup options.
-DOCKER_OPTS="--storage-driver=aufs"
-```
+    # Use DOCKER_OPTS to modify the daemon startup options.
+    DOCKER_OPTS="--storage-driver=aufs"
```

Comment:
We should have a follow up PR to be more generic, and not use `DOCKER_OPTS` here; most systems now use systemd, and we don't use `DOCKER_OPTS` on systemd